Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/BlendColorGradingLuts.azsl

```diff
@@ -116,9 +95,9 @@ void MainCS(uint3 dispatch_id: SV_DispatchThreadID)
 
     // Get coordinates within the blended LUT 3D texture
     float3 baseCoord = float3 (
-        (float)(dispatch_id.x)/(float)PassSrg::m_blendedLutDimensions.x,
-        (float)(dispatch_id.y)/(float)PassSrg::m_blendedLutDimensions.y,
-        (float)(dispatch_id.z)/(float)PassSrg::m_blendedLutDimensions.z
+        float(outPixel.x) / float(PassSrg::m_blendedLutDimensions.x - 1),
```

Comment:
Was this the primary source of the error in your screenshots?

Reply:
The two bug fixes called out in the description were both significant causes of banding because they both caused precision loss in the lut. Additionally we used to always use a log2 1000 nit shaper in the blend which would also cause precision loss (and banding) if your source lut was not using the same shaper.